soapui/src/main/java/com/eviware/soapui/tools/AbstractSoapUIRunner.java

```diff
@@ -74,18 +76,36 @@ public AbstractSoapUIRunner(String title) {
 
     protected void initGroovyLog() {
         if (!groovyLogInitialized) {
-            Logger logger = Logger.getLogger("groovy.log");
-
-            ConsoleAppender appender = new ConsoleAppender();
-            appender.setWriter(new OutputStreamWriter(System.out));
-            appender.setLayout(new PatternLayout("%d{ABSOLUTE} %-5p [%c{1}] %m%n"));
-            logger.addAppender(appender);
-
+            ensureConsoleAppenderIsDefined(Logger.getLogger("groovy.log"));
             groovyLogInitialized = true;
         }
     }
 
     /**
+     * Ensure there is one (and only one) ConsoleAppender instance configured for <code>logger</code>.
+     *
+     * @param logger
+     */
+    protected void ensureConsoleAppenderIsDefined(Logger logger) {
+        if (logger != null) {
+            // ensure there is a ConsoleAppender defined, adding one if necessary
+            ConsoleAppender consoleAppender = null;
+            for (Object appender : Collections.list(logger.getAllAppenders())) {
+                if (appender instanceof ConsoleAppender) {
+                    consoleAppender = (ConsoleAppender) appender;
+                    break;

```

Comment:
It would be even better to just return here, wouldn't it? Then you don't need the if statement on line 99 and can define the consoleAppender variable on the same line where you initialize it (currently on line 100).